Src/FluentAssertions/Primitives/StringWildcardMatchingStrategy.cs

```diff
@@ -41,11 +41,9 @@ private bool IsMatch(string subject, string expected)
 
     private static string ConvertWildcardToRegEx(string wildcardExpression)
     {
-        return "^"
-            + Regex.Escape(wildcardExpression)
-                .Replace("\\*", ".*", StringComparison.Ordinal)
-                .Replace("\\?", ".", StringComparison.Ordinal)
-            + "$";
+        return $"^{Regex.Escape(wildcardExpression)
```

Comment:
🤔 I don't think this will make it necessarily better.

Reply:
True, hadn't looked at the actual coverage